File: benchmark/results/python-multilang/run-003/a-mcp/output/inventory/services/reservation_service.py
from typing import List, Optional
from inventory.aggregates.stock_item import StockItem

"""ReservationService service."""
class ReservationService:
    def create(self, input: StockItem) -> StockItem:
        raise NotImplementedError('not implemented')
    def findById(self, id: str) -> Optional[StockItem]:
        raise NotImplementedError('not implemented')
    def list(self, limit: float) -> List[StockItem]:
        raise NotImplementedError('not implemented')
    def delete(self, id: str) -> None:
        raise NotImplementedError('not implemented')

File: benchmark/results/java-invocation-inline/run-004/b-baseline/output/com/metaengine/demo/inventory/services/ReservationService.java
package com.metaengine.demo.inventory.services;

import java.util.List;
import com.metaengine.demo.inventory.aggregates.StockItem;

/** ReservationService domain service. */
public class ReservationService {
    /** Create a Reservation against a StockItem. */
    public StockItem create(StockItem input) {
        throw new UnsupportedOperationException("not implemented");
    }

    /** Find a StockItem by id; returns null if not found. */
    public StockItem findById(String id) {
        throw new UnsupportedOperationException("not implemented");
    }

    /** List up to limit StockItems. */
    public List<StockItem> list(int limit) {
        throw new UnsupportedOperationException("not implemented");
    }

    /** Cancel a reservation by id. */
    public void delete(String id) {
        throw new UnsupportedOperationException("not implemented");
    }
}
